CMakeLists.txt

```diff
@@ -768,7 +768,9 @@ endif()
 if(ENABLE_SCRIPTING)
 	list(APPEND ENABLES SCRIPTING)
 	find_feature(USE_JSON_C "json-c")
-	if(NOT USE_LUA VERSION_LESS 5.1)
+	if(USE_LUA STREQUAL "JIT")
+		find_feature(USE_LUA "Lua${USE_LUA}")
```

Comment:
Might as well just write `LuaJIT` here since it's guaranteed to evaluate to that after the previous line.